kernel/contracts/KernelStakes.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.4.18;
+
+import "./ZepToken.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract KernelStakes is  Ownable {
```

Comment:
Just an idea, given this contract is the one who process staking and unstaking actions, what about `KernelStaker`?

Reply:
Hmm I'd understand the "staker" to be "the person who stakes", not the class that manages it